src/bookshelf/Makefile

```diff
@@ -28,7 +28,7 @@ include Makefile.db
 
 ## TODO: put 'ct' back in here
 all: $(REBAR3) elvis
-	@$(REBAR3) do clean, compile, eunit, dialyzer, xref
+	@$(REBAR3) do clean, compile, eunit, dialyzer #, xref
```

Comment:
Why did we need to remove xref?

Reply:
That was removed just for development purposes.  I often shut off Dialyzer, xref, etc. during development because they get in my way and cause issues and headaches, but I restore once the development work is done.

Reply:
xref re-enabled.